roms_tools/setup/cdr_forcing.py

```diff
@@ -93,22 +131,26 @@ def check_unique_name(self) -> "ReleaseCollector":
 
         return self
 
-    def determine_release_type(self) -> "ReleaseCollector":
+    @model_validator(mode="after")
+    def check_all_releases_same_type(self):
         """Ensure all releases are of the same type, and set the release_type."""
-        release_types = set(map(type, self.releases))
+        release_types = set(map(type, self.root))
         if len(release_types) > 1:
             type_list = ", ".join(map(str, release_types))
             raise ValueError(
                 f"Not all releases have the same type. Received: {type_list}"
             )
+        return self
 
-        return release_types.pop()
+    @property
+    def release_type(self):
+        return self.root[0].release_type
```

Comment:
```suggestion
    release_types = set(map(type, self.root))
    return release_types.pop()
```

 Helps our future selves to avoid asking what's special about the 0-th item, see also [this comment](https://github.com/CWorthy-ocean/roms-tools/pull/301#discussion_r2096590809).